### DISCUSSION TITLE
TreeView: Combine lazy loading of child nodes and recursive tree

### DISCUSSION THREAD

**Discussion by wokis** [vmware/clarity#5598] [Q&A] [answered]
Hello,

Is it possible to combine lazy loading of child nodes and recursive tree on the tree view angular component? I can't exactly wrap my head around how to do that and there is no clear example. Maybe it's not possible and partly why VMWare vCenter uses the Telerik Kendo UI component for their tree view?

Is it possible and could someone provide some example code or pointers?

**Comment by gnomeontherun** [comment, marked as answer]
This is certainly possible, and our demo here shows a lazy recursive tree. https://v4.clarity.design/documentation/tree-view#lazy-loading-recursive

**Reply by wokis** [reply]
Yeah, I found that documentation too a few minutes ago. Wonder why it disappeared from https://clarity.design/angular-components/tree-view?

**Reply by wokis** [reply]
Thanks anyway for the response :)

**Reply by gnomeontherun** [reply]
Good question, it must be a mistake we should get that back up.
